### PR TITLE
Elevator sound

### LINKED ISSUE #901: Elevator scary sound script
Create a script that plays a sound that gives the impression of something hitting the elevator.

### DoD
A sound that sounds like an elevator being hit will play with a script.

### PR DESCRIPTION
**Pull request by ludvigarlebrink**
resolves #901

### CODE REVIEW

File: Game/Resources/elevatorScene/Scripts/LightsOutScript.as [hunk 1]
```diff
@@ -38,6 +42,8 @@ class LightsOutScript {
     }
 
     void Toggle(int index) {
+        buttonSound.Stop();
```
Comment (NiceGuyAndersson) [inline Game/Resources/elevatorScene/Scripts/LightsOutScript.as]: Stop and Play only set flags. The flag set by Stop is immediately negated by Play, making the Stop call unnecessary.

File: Game/Resources.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-	"activeScene" : "Resources/Super Scene/Super Scene",
+	"activeScene" : "Resources/elevatorScene/elevatorScene_01",
```
Comment (NiceGuyAndersson) [inline Game/Resources.json]: Would prefer if the startup-scene always is Super Scene on master.

Comment (Chainsawkitten) [reply Game/Resources.json]: This is not the startup scene, but the active scene (scene open in editor).

Comment (NiceGuyAndersson) [reply Game/Resources.json]: That's what I meant, starting up the editor ;)

**Comment by ludvigarlebrink** [comment]
@andens File is removed.

**Comment by NinjaDanz3r** [comment]
iamheman

**Comment by NinjaDanz3r** [comment]
iamheman

**Pull request merged**